three_dof_planar_manipulator:
    # Publish all joint states -----------------------------------
    joints_update:
      type: joint_state_controller/JointStateController
      publish_rate: 50
    
    # Position Controllers ---------------------------------------
    joint1:
      type: position_controllers/JointPositionController
      joint: joint1
      pid: {p: 1.0, i: 1.0, d: 0.0}
    joint2:
      type: position_controllers/JointPositionController
      joint: joint2
      pid: {p: 1.0, i: 1.0, d: 0.0}
    joint3:
      type: position_controllers/JointPositionController
      joint: joint3
      pid: {p: 1.0, i: 1.0, d: 0.0}
